Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
> Maybe node-gyp used to be included with npm and no longer is or something like that?

It's present in, at least under the current rewrite here, `libexec/npm/bin/node-gyp-bin`.